frontend/src/component/project/Project/ProjectHealth/ReportTable/ReportCard/ReportCard.tsx

```diff
@@ -149,7 +174,9 @@ export const ReportCard = ({ healthReport }: IReportCardProps) => {
                 </StyledList>
             </Box>
             <Box sx={{ flexBasis: '40%' }}>
-                <StyledHeader>Potential actions</StyledHeader>
+                <StyledHeader>
+                    Potential actions <span>{renderStalenessInfoIcon()}</span>
```

Comment:
I suggest to use a component here instead of a function call <StalenessInfoIcon>